nova_vm/src/ecmascript/builtins/array_buffer/abstract_operations.rs

```diff
@@ -422,29 +447,40 @@ pub(crate) fn numeric_to_raw_bytes(
 /// type (a TypedArray element type), value (a Number or a BigInt),
 /// isTypedArray (a Boolean), and order (SEQ-CST, UNORDERED, or INIT) and
 /// optional argument isLittleEndian (a Boolean) and returns UNUSED.
-pub(crate) fn set_value_in_buffer(
-    _array_buffer: ArrayBuffer,
-    _byte_index: u32,
-    _type: (),
-    _value: Value,
+#[allow(clippy::too_many_arguments)]
+pub(crate) fn set_value_in_buffer<T: Viewable>(
+    agent: &mut Agent,
+    gc: GcScope<'_, '_>,
```

Comment:
**note:** Same thing here, this cannot trigger GC.

Reply:
Required because the `to_*` conversion functions called by `from_le_value` and `from_be_value` require the GC token.